tools/deployment/make_linux_package.sh

```diff
@@ -117,6 +117,7 @@ function main() {
   platform OS
   distro $OS DISTRO
 
+  touch "$BUILD_DIR/$PACKAGE_NAME$(get_pkg_suffix)"
```

Comment:
Ah! Understood, perhaps the `realpath` output varies: 
```
/tmp » stat /tmp/hello_friend    
stat: cannot stat '/tmp/hello_friend': No such file or directory
/tmp » realpath /tmp/hello_friend
/tmp/hello_friend
```

How about `realpath`ing the directory, and then appending the filename. We don't need the relative path, but it help with tooling that parses the job output.